tests/eval_files/trio102.py

```diff
@@ -120,7 +120,9 @@ async def foo():
     try:
         pass
     finally:
-        async for i in trio.bypasslinters:  # error: 8, Statement("try/finally", lineno-3)
+        async for (  # error: 8, Statement("try/finally", lineno-3)
+            i
+        ) in trio.bypasslinters:
```

Comment:
What spurred this?

Reply:
New version of Black now considers the line too long (90>88 after all) and splits it up, after which I move the comment back to line 123.
Not super pretty, maybe worth reporting upstream.